app/event_source/subscribers/families/find_by_requested_subscriber.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Subscribers
+  module Families
+    # Subscriber will receive a request(from edi_gateway) to find family
+    class FindByRequestedSubscriber
+      include ::EventSource::Subscriber[amqp: 'edi_gateway.families']
+
+      subscribe(:on_find_by_requested) do |delivery_info, _properties, response|
+        logger.info "on_find_by_requested response: #{response}"
+        subscriber_logger = subscriber_logger_for(:on_families_find_by_requested)
+        response = JSON.parse(response, symbolize_names: true)
+        logger.info "on_find_by_requested response: #{response}"
+        subscriber_logger.info "on_find_by_requested response: #{response}"
+
+        process_find_by_requested_event(subscriber_logger, response) unless Rails.env.test?
+
+        ack(delivery_info.delivery_tag)
+      rescue StandardError, SystemStackError => e
+        logger.info "on_find_by_requested error: #{e} backtrace: #{e.backtrace}; acked (nacked)"
+        ack(delivery_info.delivery_tag)
+      end
+
+      private
+
+      def error_messages(result)
+        if result.failure.is_a?(Dry::Validation::Result)
+          result.failure.errors.to_h
+        else
+          result.failure
+        end
+      end
+
+      def process_find_by_requested_event(subscriber_logger, response)
+        subscriber_logger.info "process_find_by_requested_event: ------- start"
+        result = ::Operations::Families::FindBy.new.call(response)
+
+        if result.success?
+          message = result.success
+          subscriber_logger.info "on_find_by_requested acked #{message.is_a?(Hash) ? message[:event] : message}"
+        else
+          subscriber_logger.info "process_find_by_requested_event: failure: #{error_messages(result)}"
+        end
+        subscriber_logger.info "process_find_by_requested_event: ------- end"
+      rescue StandardError => e
+        subscriber_logger.info "process_find_by_requested_event: error: #{e} backtrace: #{e.backtrace}"
```

Comment:
I think this should be `subscriber_logger.error` (there is another one on the subscribe), it's easier to search on greylog